File: P1A5/P3.java
class Fecha {

    public int mes;
    public int dia;
    public int anio;

    public Fecha(int mes, int dia, int anio) {
        this.mes = mes;
        this.dia = dia;
        this.anio = anio;
    }

    public void mostrarFecha() {
        System.out.println(mes + "/" + dia + "/" + anio);
    }
}

public class Main {

    public static void main(String[] args) {

        Fecha fecha1 = new Fecha(8, 30, 2026);
        Fecha fecha2 = new Fecha(12, 25, 2026);

        System.out.println("Fecha 1:");
        fecha1.mostrarFecha();

        System.out.println("Fecha 2:");
        fecha2.mostrarFecha();
    }
}

File: P1A5/P3.cpp
#include <iostream>

using namespace std;

class Fecha {

public:

    int mes;
    int dia;
    int anio;

    Fecha(int mes, int dia, int anio) {
        this->mes = mes;
        this->dia = dia;
        this->anio = anio;
    }

    void mostrarFecha() {
        cout << mes << "/" << dia << "/" << anio << endl;
    }
};

int main() {

    Fecha fecha1(8, 30, 2026);
    Fecha fecha2(12, 25, 2026);

    cout << "Fecha 1:" << endl;
    fecha1.mostrarFecha();

    cout << "Fecha 2:" << endl;
    fecha2.mostrarFecha();

    return 0;
}